database/loadVolumeData.js

```diff
@@ -0,0 +1,243 @@
+    if (err) {
+      callback(err)
+    } else {
+      logInColour('success inserting policies for team ' + teamId)
+
+      if (teamId < NUM_TEAMS + TEAM_START_ID - 1) {
+        // load policies for next team
+        loadPolicies(callback, id, orgId, teamId + 1)
+      } else {
+        // move on to loading users
+        loadUsers(callback, USER_START_ID, orgId, TEAM_START_ID)
+      }
+    }
+  })
+}
+
+// insert users and add them to teams in batches
+function loadUsers (callback, startId, orgId, teamId) {
+  // insert users
+  console.log('inserting users ' + startId + ' to ' + (startId + NUM_USERS_PER_TEAM - 1) + ' into team: ' + teamId)
+
+  var userSql = 'INSERT INTO users (id, name, org_id)\nVALUES\n'
+  var userTeamSql = 'INSERT INTO team_members (user_id, team_id)\nVALUES\n'
+  for (var id = startId; id < (startId + NUM_USERS_PER_TEAM); id++) {
+    userSql += "('" + id + "', 'USER_" + id + "', '" + orgId + "')"
+    userTeamSql += "('" + id + "', '" + teamId + "')"
+
+    if (id === startId + NUM_USERS_PER_TEAM - 1) {
+      userSql += ';'
+      userTeamSql += ';'
+    } else {
+      userSql += ',\n'
+      userTeamSql += ',\n'
+    }
+  }
+
+  var fixturesSQL = 'BEGIN;\n'
+  fixturesSQL += userSql + '\n'
+  fixturesSQL += userTeamSql + '\n'
+  fixturesSQL += 'COMMIT;\n'
+
+  client.query(fixturesSQL, function (err, result) {
+    if (err) {
+      callback(err)
+    } else {
+      logInColour('success inserting users ' + startId + ' to ' + (startId + NUM_USERS_PER_TEAM - 1))
+      if (teamId < NUM_TEAMS + TEAM_START_ID - 1) {
+        loadUsers(callback, id, orgId, teamId + 1)
+      } else {
+        loadVolumeDataEnd(callback)
+      }
+    }
+  })
+}
+
+function logInColour (message, level = 'success') {
+  var colour = '\x1b[0m'
+  var reset = '\x1b[0m'
+  if (level === 'success') {
+    colour = '\x1b[32m'
+  } else if (level === 'error') {
+    colour = '\x1b[31m'
+  }
+  console.log(colour, message, reset)
+}
+
+function loadVolumeDataEnd (callback) {
+  /* moved to migration scripts
+  if (CREATE_INDEX === true) {
+    console.log('Creating index, please wait...')
+    var sql = 'CREATE INDEX "team_members#user_id"\n' +
+      'ON public.team_members USING btree\n' +
+      '(user_id COLLATE pg_catalog."default" varchar_ops)\n' +
+      'TABLESPACE pg_default;\n\n' +
+      'ALTER TABLE public.team_members\n' +
+    'CLUSTER ON "team_members#user_id;"\n'
+
+    console.log(sql)
+
+    // creating index doesn't seem to want to call back...
+    client.query(sql, function (err, result) {
+      if (err) {
+        logInColour('Error creating index: ' + err + ', result:' + result, 'error')
+        callback(err)
+      } else {
+        logInColour('success creating index' + result)
+        callback(null)
+        return
+      }
+    })
+  }
+  */
+  endTime = Date.now()
+  logInColour('loadVolumeData completed in ' + (endTime - startTime) + 'ms')
+
+  callback(null) // done
```

Comment:
no need for `null` here, just `callback()`

Reply:
think i fixed all those, thanks Damien